package.json

```diff
@@ -4,9 +4,10 @@
   "description": "TypeScript / JavaScript OPC UA client for the browser",
   "main": "index.js",
   "scripts": {
-    "test": "tsc && jest",
-    "tsc": "tsc --skipLibCheck",
-    "watch": "tsc -w --skipLibCheck",
+    "clear": "rm -fr dist coverage",
+    "test": "npm run clear && tsc --sourceMap && jest",
+    "tsc": "npm run clear && tsc --skipLibCheck",
+    "watch": "npm run clear && tsc -w --skipLibCheck",
```

Comment:
I'd like to have two separate scripts. One just for testing and one also for coverage. I want coverage during CI but I don't want coverage every time I run the tests. Especially at the moment where a lot of stuff is changing I just want to see the tests. The coverage puts a lot of noise on the command line output.

Reply:
See if the scripts fit that way